### ISSUE TITLE
Request: Sort for category display

### ISSUE THREAD

**Issue by zombiewoof** [thinkle/gourmet#838]
It would be nice if the recipes, in each category, could be sorted for the display. Currently, when the recipes in a category display, they display in the order that they were added to the database. Sorted by name would make it easier to find a particular recipe. If not a mandatory sort (as with the categories themselves), at least an option to sort with the ability to save the setting would be appreciated.

**Comment by thinkle** [comment]
Click any column header and it should sort by that field.

**Comment by allenerenee** [comment]
To clarify, you can sort by multiple columns at a time by doing so, in the
order that you select them.

**Comment by zombiewoof** [comment]
On Thu, 2016-01-21 at 07:59 -0800, Tom Hinkle wrote:
> Click any column header and it should sort by that field.

That is correct, in the Search Recipes view. I'm referring to the Browse
Recipes-->Category-->Whatever Category view.

Say, for instance, that I have a category named 'Seafood' and I go to
Browse Recipes-->Category-->Seafood. It would be nice to have the recipe
icons display in alphabetical order instead of the order entered into
the database. Doesn't matter if it is set with code, as the icons for
the categories are, or with a setting that the user can choose - but it
would be good if the choice would 'stick' (be saved for the next time
the program runs).
-- 
Mike Strickland
LivinWell@GeorgiaNatives.net
---------------------------
Homepage: A Rescuer's Guide to Georgia Native Plants
        GeorgiaNatives.net

**Comment by thinkle** [comment]
Ah -- gotcha. Haven't played with the browse interface in a while, but that
enhancement makes sense and should be easy to make.

**Comment by zombiewoof** [comment]
On Thu, 2016-01-21 at 13:14 -0800, Tom Hinkle wrote:
> Ah -- gotcha. Haven't played with the browse interface in a while, but
> that
> enhancement makes sense and should be easy to make.

I think that alphabetical ordering would be good for all recipe groups
viewed under the Browse Recipes section - I mostly use Category, so
that's what I used as an example.
-- 
Mike Strickland
LivinWell@GeorgiaNatives.net
---------------------------
Homepage: A Rescuer's Guide to Georgia Native Plants
        GeorgiaNatives.net

**Comment by zombiewoof** [comment]
I applied in the patch, provided by googlymog, but all it did was remove the browse tab from the main window. I have reverted to the original browser.py.

**Comment by zombiewoof** [comment]
Since the failed patch attempt, back in July, I have upgraded to Ubuntu 16.04 (clean install), and that entailed an upgrade to Gourmet (to 0.17.4), from the repository. I have applied the patch provided by googlymog and it works like a charm :-)

I don't know why it wouldn't work with the older version of the browse script.

I would recommend that the patch be applied to the main code, so that all will benefit.
